assets/wizards/newspack/views/settings/connections/google-oauth.tsx

```diff
@@ -0,0 +1,169 @@
+/**
+ * Settings Wizard: Connections > Google OAuth
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies.
+ */
+import { Button } from '../../../../../components/src';
+import WizardsActionCard from '../../../../wizards-action-card';
+import { useWizardApiFetch } from '../../../../hooks/use-wizard-api-fetch';
+import { WizardError, WIZARD_ERROR_MESSAGES } from '../../../../errors';
+
+function getURLParams() {
```

Comment:
Non-blocking question: Can we use react routers [useSearchParams](https://reactrouter.com/en/main/hooks/use-search-params) or [useParams](https://reactrouter.com/en/main/hooks/use-params) here?

Reply:
Thanks for the suggestion @chickenn00dle !

- `useSearchParams` is only available in react-router-dom v6, and we are currently using v5.
- `useParams` handles path parameters, not query parameters.

Given our current setup, using a custom function like getURLParams is straightforward and avoids adding unnecessary complexity or dependencies.